lib/trino-filesystem-azure/src/main/java/io/trino/filesystem/azure/AzureOutputFile.java

```diff
@@ -69,9 +69,7 @@ public OutputStream create(AggregatedMemoryContext memoryContext)
     public void createOrOverwrite(byte[] data)
             throws IOException
     {
-        try (OutputStream out = createOutputStream(newSimpleAggregatedMemoryContext(), true)) {
-            out.write(data);
-        }
+        blobClient.upload(BinaryData.fromBytes(data), true);
```

Comment:
should we consider `blobClient.getBlockBlobClient().upload(BinaryData.fromBytes(data), true);` instead as below gets stream from `BlockBlobClient`, may be Trino operates on Block blobs.

https://github.com/trinodb/trino/blob/e7aa58955b0578ab8e860b9ff44b9b65e45f084a/lib/trino-filesystem-azure/src/main/java/io/trino/filesystem/azure/AzureOutputStream.java#L76

Reply:
Thanks! I had to go and read [the docs](https://learn.microsoft.com/en-us/rest/api/storageservices/understanding-block-blobs--append-blobs--and-page-blobs) to learn about different blob types. It certainly is better to be explicit, especially since the blob type cannot be changed after creating it.